src/main/src/main/java/com/shevtsov/validators/impl/ValidationServiceImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.shevtsov.validators.impl;
+
+import com.shevtsov.dao.ClientDao;
+import com.shevtsov.dao.impl.ClientDaoImpl;
+import com.shevtsov.exceptions.BusinessException;
+import com.shevtsov.validators.ValidationService;
+
+import java.util.Arrays;
+
+public class ValidationServiceImpl implements ValidationService {
+    private final ClientDao clientDao = ClientDaoImpl.getInstance();
+
+    @Override
+    public void validateAge(int age) throws BusinessException {
+        if (age < 0 || age > 200) {
+            throw new BusinessException("Invalid age!!!");
+        }
+    }
+
+    @Override
+    public void validatePhoneFormat(String phone) throws BusinessException {
+        String[] operatorCodes = {"067", "097", "050"};
```

Comment:
I think this can be static to avoid create new one each time when method call

Reply:
done